src/theory/sets/term_registry.cpp

```diff
@@ -152,6 +154,21 @@ void TermRegistry::debugPrintSet(Node s, const char* c) const
   }
 }
 
+void TermRegistry::sendSimpleLemmaInternal(Node eq, InferenceId id)
```

Comment:
The node argument here could be an equality node, or a singleton node. Please rename the argument to `n`